### PR TITLE
chore: Code restructuring. Move maven plugin into top level directory

### PR DESCRIPTION
**Pull request by kappratiksha**
Move function-maven-plugin out of invoker into the top level directory.

### CODE REVIEW

**Review by anniefu — APPROVED** [review]

File: function-maven-plugin/pom.xml [hunk 1]
```diff
@@ -44,7 +45,7 @@
     <dependency>
       <groupId>com.google.cloud.functions.invoker</groupId>
       <artifactId>java-function-invoker</artifactId>
-      <version>1.1.2-SNAPSHOT</version>
+      <version>1.1.1</version>
```
Comment (jihuin) [inline function-maven-plugin/pom.xml:48]: I guess this version bump is done by release-prepare while testing? We should probably revert back

File: function-maven-plugin/pom.xml [hunk 2]
```diff
@@ -85,4 +86,52 @@
       </plugin>
     </plugins>
   </build>
+    <distributionManagement>
```
Comment (jihuin) [inline function-maven-plugin/pom.xml:89]: nit: indentation is inconsistent

**Review by jihuin — APPROVED** [review]

File: function-maven-plugin/pom.xml (hunk 1 above)
Comment (kappratiksha) [reply function-maven-plugin/pom.xml:48]: This is not done by release-prepare. I added this to remove the interdependency between function-maven-plugin and invoker.

Comment (jihuin) [reply function-maven-plugin/pom.xml:48]: Got it. Thanks for clarification.

**Pull request merged**